CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.5.1 (06/01/2016)
+
+### Changes
+
+* Change header name emitted by `Client#apply` from "Authorization" to "authorization" ([@murgatroid99][])
+
```

Comment:
can you add the other changes between 0.5.0 and this one? I think there is at least one more.